src/devices/Common/tests/ValueArrayTests.cs

```diff
@@ -40,6 +40,7 @@ public void AssigningDefaultClearsArray()
             Assert.Equal(0, _myArray[2]);
         }
 
+#if NET5_0_OR_GREATER
```

Comment:
honestly I think we should just not build tests against lower versions

Reply:
(nothing to change in this PR)

Reply:
There are some test assemblies that explicitly build against .NET4.8, just to test the netstandard2.0 behaviors, where they are different.